Program.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+
+namespace Homework3
+{
+    /// <summary>
+    /// Represents a logger for logging messages to a file.
+    /// </summary>
+    class Logger : IDisposable
+    {
+        private static Logger instance;
```

Comment:
Name of logger instance should be _logger or something more resonable. Question reads as "Include a private static field instance of type Logger" which means to create an instance of Logger not to name it as instance.